bootstrap/components/sinks/kafka-sink-topology-component.json

```diff
@@ -35,13 +35,23 @@
         "hint": "schema,override"
       },
       {
-        "uiName": "Writer schema version",
+        "uiName": "Writer Schema Branch",
+        "fieldName": "schemaBranch",
+        "isOptional": false,
+        "tooltip": "Schema branch to be used fetching for writer schema versions. Default branch is MASTER",
+        "type": "enumstring",
+        "options": [],
+        "hint": "dependOn-topic,schemaBranch",
+        "defaultValue": "MASTER"
```

Comment:
@satishd due to this "defaultValue", bootstrap.sh is failing to add source/sink. We have hardcode the default value of branch to "MASTER" in UI so if you want, can remove this field or please have a look why it's failing.

Reply:
@satishd @shahsank3t Its failing because its expecting the default value to be one of the items from the options. But in this case because options is an empty list which is populated later by making an API call, its throwing an error. We can just remove the default value in the json spec since thats how other such fields work.

Reply:
@priyank5485 @shahsank3t Right, we can remove default values for this mandatory field.